### PR TITLE
[10.3] Implement Runners API

### PR DESCRIPTION
**Pull request by WaylandAce**
Related to #339

### CODE REVIEW

**Comment by m1guelpf** [comment]
is this ready for merge?

**Comment by WaylandAce** [comment]
Not yet - not all runners api implemented. I'll remove prefix WIP, when it will be ready

**Comment by GrahamCampbell** [comment]
Thanks for starting work on this @WaylandAce. Are you able to finish this PR? 🚀

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 vendor/
 composer.lock
+.idea/
```
Comment (GrahamCampbell) [inline .gitignore:3]: Please don't add this.

File: lib/Gitlab/Api/Runners.php [hunk 2]
```diff
@@ -0,0 +1,149 @@
+<?php namespace Gitlab\Api;
```
Comment (GrahamCampbell) [inline lib/Gitlab/Api/Runners.php:1]: Missing file copyright header.

File: test/Gitlab/Tests/Api/ProjectsTest.php [hunk 3]
```diff
@@ -40,8 +40,7 @@ public function shouldNotNeedPaginationWhenGettingProjects()
         $api->expects($this->once())
             ->method('get')
             ->with('projects')
-            ->will($this->returnValue($expectedArray))
-        ;
+            ->will($this->returnValue($expectedArray));
```
Comment (GrahamCampbell) [inline test/Gitlab/Tests/Api/ProjectsTest.php:43]: Please revert all the changes to this file.

**Comment by GrahamCampbell** [comment]
Ping @WaylandAce.

**Comment by GrahamCampbell** [comment]
Closing due to inactivity. I don't have capacity to take this on and finish up. I would, of course, review a new PR with appropriate corrections.

**Pull request closed without merging**